.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: scala
 scala:
-   - 2.10.5
-   - 2.11.7
+   - 2.10.6
+   - 2.11.8
```

Comment:
Seems like this conflicts with #221, maybe merge the PRs into a single one? Or did you want one of them reviewed first?